base/services/w32time/register.c

```diff
@@ -236,7 +236,7 @@ SetNtpClientValues(VOID)
         goto done;
     }
 
-    dwValue = 0x00093a80;
+    dwValue = 4 * 60 * 60; // 4 hour poll time for setting clock
```

Comment:
What I do wonder is: If Windows actually uses a longer interval, but still when DST-changes it does manage to update that right after just minutes or an hour! That means that there must be some explicit trigger on Windows on DST-changes. That seems to be a different trigger than the regular interval-trigger that you touched here.

Reply:
I recommend the following test: Do disable the w32timesvc on WInows, and see whether it still does perform correctly the DST changes. Maybe it's not even the services job to do that by online sync? But some other part may be responsible for that? (with question mark!)
After all: Windows could already do properly DST switch, when many peoples Windows 95 had no internet connection at all.
Maybe we do abuse the service here, for something else should be responsible for? (with question mark!)

Reply:
@Doug-Lyons Please consider @SergeGautherie 's suggestion here (regarding having a `#define` for it, most probably in w32time main header), since now we have two places in w32time where this default fallback value of "4 * 60 * 60" is hardcoded.

Reply:
New value of W32TIME_POLL_INTERVAL now defined in w32time.h. Thanks.